### PR TITLE
Add functional test.

### PR DESCRIPTION
**Pull request by YuriGizra**
#2

### CODE REVIEW

File: behat/features/homepage.feature [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Feature: Homepage
-  In order to be able to view and get info about the site
+  In order to be able to view and get info about a deal.
   As an anonymous user
-  We need to be able to have access to the homepage
+  We need to be able to select a deal and get access to order flight
 
-  @api
-  Scenario: Visit the homepage
+  @api @javascript
```
Comment (amitaibu) [inline behat/features/homepage.feature]: remove `@api`

File: behat/features/homepage.feature [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+  Scenario: Visit the asia deals
```
Comment (amitaibu) [inline behat/features/homepage.feature]: asia -> Asia

File: behat/features/homepage.feature [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
     Given I am an anonymous user
-    When  I visit the homepage
-    Then  I should have access to the page
+    When  I go to "http://www.elal.com/en/Deals/Offers/Asia/Pages/Hong-Kong-Beijing.aspx"
+    And   I select deal
```
Comment (amitaibu) [inline behat/features/homepage.feature]: => `And I select a deal`

**Comment by YuriGizra** [comment]
@amitaibu, fixed, ready for review.

**Comment by YuriGizra** [comment]
Merged manually.

**Pull request closed without merging**